common/zoom/zoom-scheduled-meeting.ts

```diff
@@ -115,10 +112,10 @@ async function getZoomMeeting(meetingId: string) {
         1000
     );
 
-    return response.json() as unknown as ZoomMeeting;
+    return (await response.json()) as ZoomMeeting;
```

Comment:
response.json() does not return a ZoomMeeting, it returns a Promise<ZoomMeeting> ...

Reply:
Also it returns a single zoom meeting and not a list of zoom meetings?

Reply:
It really returns only **one** meeting with this shape 

```json
{
  "assistant_id": "kFFvsJc-Q1OSxaJQLvaa_A",
  "host_email": "jchill@example.com",
  "host_id": "30R7kT7bTIKSNUFEuH_Qlg",
  "id": 97763643886,
  "uuid": "aDYlohsHRtCd4ii1uC2+hA==",
  "agenda": "My Meeting",
  "created_at": "2022-03-25T07:29:29Z",
  "duration": 60,
  "encrypted_password": "8pEkRweVXPV3Ob2KJYgFTRlDtl1gSn.1",
  "h323_password": "123456",
  "join_url": "https://example.com/j/11111",
  "occurrences": [
    {
      "duration": 60,
      "occurrence_id": "1648194360000",
      "start_time": "2022-03-25T07:46:00Z",
      "status": "available"
    }
  ],
  "password": "123456",
  "pmi": "97891943927",
  "pre_schedule": false,
  "recurrence": {
    "end_date_time": "2022-04-02T15:59:00Z",
    "end_times": 7,
    "monthly_day": 1,
    "monthly_week": 1,
    "monthly_week_day": 1,
    "repeat_interval": 1,
    "type": 1,
    "weekly_days": "1"
  },
  "settings": {
    "allow_multiple_devices": true,
    "alternative_hosts": "jchill@example.com;thill@example.com",
    "alternative_hosts_email_notification": true,
    "alternative_host_update_polls": true,
    "approval_type": 0,
    "approved_or_denied_countries_or_regions": {
      "approved_list": [
        "CX"
      ],
      "denied_list": [
        "CA"
      ],
      "enable": true,
      "method": "approve"
    },
    "audio": "telephony",
    "audio_conference_info": "test",
    "authentication_domains": "example.com",
    "authentication_exception": [
      {
        "email": "jchill@example.com",
        "name": "Jill Chill",
        "join_url": "https://example.com/s/11111"
      }
    ],
    "authentication_name": "Sign in to Zoom",
    "authentication_option": "signIn_D8cJuqWVQ623CI4Q8yQK0Q",
    "auto_recording": "cloud",
    "breakout_room": {
      "enable": true,
      "rooms": [
        {
          "name": "room1",
          "participants": [
            "jchill@example.com"
          ]
        }
      ]
    },
    "calendar_type": 1,
    "close_registration": false,
    "contact_email": "jchill@example.com",
    "contact_name": "Jill Chill",
    "custom_keys": [
      {
        "key": "key1",
        "value": "value1"
      }
    ],
    "email_notification": true,
    "encryption_type": "enhanced_encryption",
    "focus_mode": true,
    "global_dial_in_countries": [
      "US"
    ],
    "global_dial_in_numbers": [
      {
        "city": "New York",
        "country": "US",
        "country_name": "US",
        "number": "+1 1000200200",
        "type": "toll"
      }
    ],
    "host_video": true,
    "jbh_time": 0,
    "join_before_host": true,
    "language_interpretation": {
      "enable": true,
      "interpreters": [
        {
          "email": "interpreter@example.com",
          "languages": "US,FR"
        }
      ]
    },
    "sign_language_interpretation": {
      "enable": true,
      "interpreters": [
        {
          "email": "interpreter@example.com",
          "sign_language": "American"
        }
      ]
    },
    "meeting_authentication": true,
    "mute_upon_entry": false,
    "participant_video": false,
    "private_meeting": false,
    "registrants_confirmation_email": true,
    "registrants_email_notification": true,
    "registration_type": 1,
    "show_share_button": true,
    "use_pmi": false,
    "waiting_room": false,
    "watermark": false,
    "host_save_video_order": true
  },
  "start_time": "2022-03-25T07:29:29Z",
  "start_url": "https://example.com/s/11111",
  "status": "waiting",
  "timezone": "America/Los_Angeles",
  "topic": "My Meeting",
  "tracking_fields": [
    {
      "field": "field1",
      "value": "value1",
      "visible": true
    }
  ],
  "type": 2
}
```